Tasks/GooglePlayPromoteV2/task.loc.json

```diff
@@ -11,7 +11,7 @@
   ],
   "version": {
     "Major": "3",
-    "Minor": "201",
+    "Minor": "205",
```

Comment:
Could you please bump the tasks versions?

Reply:
Also, please bump the extension version in the following files:
- `vsts-extension-google-play.json`
- `package.json`
- `package-lock.json`

Reply:
> Could you please bump the tasks versions?
Do you mean bump **minor** versions? Seems like they are already bumped for all tasks.

Reply:
Done.